cmd/multifile/manifest.go

```diff
@@ -94,24 +86,35 @@ func parseManifest(file io.Reader) (pget.Manifest, error) {
 		// is allowed/not allowed/etc
 		consumer := viper.GetString(config.OptOutputConsumer)
 		if consumer != config.ConsumerNull {
-			err = checkSeenDests(seenDests, dest, urlString)
+			err = checkSeenDestinations(seenDestinations, dest, urlString)
 			if err != nil {
+				if errors.Is(err, errDupeURLDestCombo) {
+					logger.Warn().
+						Str("url", urlString).
+						Str("destination", dest).
+						Msg("Parse Manifest: Skip Duplicate URL/Destination")
+					continue
+				}
 				return nil, err
 			}
-			seenDests[dest] = urlString
+			seenDestinations[dest] = urlString
 
 			err = cli.EnsureDestinationNotExist(dest)
 			if err != nil {
 				return nil, err
 			}
 		}
-		schemeHost, err := client.GetSchemeHostKey(urlString)
-		if err != nil {
-			return nil, fmt.Errorf("error parsing url %s: %w", urlString, err)
-		}
+		if valid, err := validURL(urlString); !valid {
```

Comment:
should validURL be called before seenDestinations is updated.?

Reply:
Shouldn't matter order here as valid url is an error case that exits pget. If it was a non-fatal error, I think it is more reasonable to call it first. 

I am open to making it non-fatal. But that is outside the scope of this PR